lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1823,6 +1826,32 @@ SubExprInfo ExprEmitter::visitSV(ArrayIndexInOutOp op) {
   return {Selection, arrayPrec.signedness};
 }
 
+SubExprInfo ExprEmitter::visitSV(IndexedPartSelectInOutOp op) {
+  auto prec = emitSubExpr(op.input(), Selection, OOLUnary);
+  os << '[';
+  emitSubExpr(op.base(), LowestPrecedence, OOLBinary);
+  if (op.decrement())
+    os << " -";
+  else
+    os << " +";
+  os << ": " << op.width();
+  os << ']';
+  return {Selection, prec.signedness};
+}
+
+SubExprInfo ExprEmitter::visitSV(IndexedPartSelectOp op) {
+  auto info = emitSubExpr(op.input(), LowestPrecedence, OOLUnary);
+  os << '[';
+  emitSubExpr(op.base(), LowestPrecedence, OOLBinary);
```

Comment:
Did you double check the precedence?

Reply:
Just doubled checked the spec for its precedence.